app/models/role.rb

```diff
@@ -75,6 +75,27 @@ class Role < ApplicationRecord
   scoped_search :on => :builtin, :complete_value => { :true => true, :false => false }
   scoped_search :on => :description, :complete_value => false
 
+  class << self
+    attr_accessor :bypass_locking
```

Comment:
The class variable could be `modify_locked` too, and the class method `ignore_locking`. I tried doing this with using a class variable (same method as the instance), but it didn't quite work yet...